dataset_test.go

```diff
@@ -184,3 +184,24 @@ func TestDataset_Compare_mixTypes(t *testing.T) {
 	results, _ = d1Dataset.Compare(d2Dataset)
 	require.Equal(t, expected, results)
 }
+
+func TestDataset_Expand_invariant(t *testing.T) {
+	c1 := strs([]string{"a"})
+	c2 := strs([]string{"b"})
+	c3 := strs([]string{"c"})
+
+	// create ds with width 3, but column capacity of 4
```

Comment:
should it be the following?
```suggestion
	// create ds with width 3, but columns capacity of 4
```